instagram/src/App.js

```diff
@@ -1,31 +1,18 @@
 import React, { Component } from 'react';
 import './App.css';
-import dummyData from './utilities/dummy_data';
-import PostContainer from './components/PostContainer/PostContainer';
-import SearchBar from './components/SearchBar/SearchBar';
+import PostPage from './components/PostContainer/PostPage';
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      posts: dummyData,
-      searchQuery: ''
     }
   }
 
-  onSearch(searchQuery) {
-    this.setState({
-      ...this.state,
-      searchQuery
-    })
-  }
 
   render() {
     return (
-      <div className="App">
-        <SearchBar onSearch={(searchQuery) => this.onSearch(searchQuery)} />
-        <PostContainer searchQuery={this.state.searchQuery} posts={this.state.posts} />
-      </div>
+      <PostPage />
```

Comment:
Your App seems to just be rendering your PostsPage now. You could probably get rid of your App component easily without any loss of functionality or organization.